deps/triemap/triemap.c

```diff
@@ -172,7 +212,12 @@ int TrieMapNode_Add(TrieMapNode **np, char *str, tm_len_t len, void *value, Trie
     __trieMapNode_children(n)[char_offset] = child;
     return rv;
   }
-  *np = __trieMapNode_AddChild(n, str, offset, len, value);
+  
+  ptr = childKeys; 
+  while(ptr < childKeys + n->numChildren && *ptr < c) {++ptr;}
+  *np = __trieMapNode_AddChildIdx(n, str, offset, len, value, ptr - childKeys);
+  //int idx = TrieMap_BinaryScan(childKeys, c, n->numChildren);
```

Comment:
You do not use binary scan here, either use it or delete it

Reply:
In benchmarks it showed no effect when compared to `memcmp`